### PR TITLE
[FIX display] don't call gemini with empty urls

### PR DESCRIPTION
**Pull request by eessex**
Addresses https://artsyproduct.atlassian.net/browse/GROW-889

Fixes a bug where gemini resize was called with empty strings, causing long timeouts and impacting article load performance.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit f8efb7c6d0aaa0f796e61661f18d55b6f1eaa514

https://deploy-preview-1251--artsy-reaction.netlify.com

File: src/Components/Publishing/Display/Canvas/CanvasVideo.tsx [hunk 1]
```diff
@@ -134,11 +134,14 @@ export class CanvasVideo extends Component<CanvasVideoProps, any> {
 
   render() {
     const { isPlaying } = this.state
+    const { coverUrl, src } = this.props
 
-    const resized_cover = resize(this.props.coverUrl, {
-      width: 760,
-      isDisplayAd: true,
-    })
+    const resized_cover =
```
Comment (l2succes) [inline src/Components/Publishing/Display/Canvas/CanvasVideo.tsx]: 👍 @eessex we should also add a check in `resize` to make sure we never send empty urls to gemini

Comment (l2succes) [reply src/Components/Publishing/Display/Canvas/CanvasVideo.tsx]: something as simple as

```javascript
if (!url) return null
```

Here https://github.com/artsy/reaction/blob/master/src/Utils/resizer.ts#L58

**Comment by joeyAghion** [comment]
Thanks for this. I'll watch the metrics post-deploy!

This makes me think Gemini should _also_ fail much more gracefully than this. I'll ticket that and fix if straightforward.

File: src/Components/Publishing/Display/Canvas/CanvasVideo.tsx (hunk 1 above)
Comment (eessex) [reply src/Components/Publishing/Display/Canvas/CanvasVideo.tsx]: Good call will add this

**Pull request merged**